crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -138,23 +222,24 @@ mod tests {
         let input = "This is not Thin Edge JSON";
         let result = converter.try_convert(&new_tedge_message(input));
 
-        assert_matches!(result, Err(ConversionError::FromThinEdgeJsonParser(_)))
+        assert_matches!(result, Err(FromSerdeJson(_)))
```

Comment:
I think this test `try_convert_invalid_json_returns_error` test is no longer applicable as we now accepts all JSON. It looks duplicated with the test `convert_error`

Reply:
fixed